### DISCUSSION TITLE
How to increase the length of the session ID token

### DISCUSSION THREAD

**Discussion by eugenestride** [ServiceStack/Discuss#61] [Q&A]
Hi there,

With reference to https://docs.servicestack.net/sessions

Currently, the session ID token has a length of 20 characters.

I have looked through the documentation but unable to find a way easily to increase the length of the session ID token.

Anyone have any idea how to do it?
Any help will be appreciated.
Thanks in advance.

**Comment by mythz** [comment]
This wasn't overridable before, but I've just added a change where you can override the session id creation by overriding `CreateSessionId()` in your AppHost, e.g:

```csharp
public class AppHost : AppHostBase
{
    //...
    public override string CreateSessionId()
    {
        // return random string (ideally URL Safe string without '+' or '/' chars)
    }
}
```

This change is available from v6.8.1+ that's now [available on MyGet](https://docs.servicestack.net/myget)
